src/main/resources/wildfly/configure-wildfly.cli

```diff
@@ -54,3 +54,8 @@ data-source add \
 # To add request logging you can uncomment the following lines
 #/subsystem=undertow/configuration=filter/custom-filter=request-dumper:add(class-name="io.undertow.server.handlers.RequestDumpingHandler", module="io.undertow.core")
 #/subsystem=undertow/server=default-server/host=default-host/filter-ref=request-dumper:add
+
+# Create custom ZAC mail session
```

Comment:
I would move this up in the file, so that it is above the commented-out logging configurations.